embedding/csharp/Tizen.Flutter.Embedding/NUIFlutterView.cs

```diff
@@ -58,29 +56,21 @@ public bool RunEngine()
                 return false;
             }
 
-            if (Engine == null)
-            {
-                Engine = new FlutterEngine();
-            }
-
+            Engine = Engine ?? new FlutterEngine();
```

Comment:
nit: There's another occurrence of this in `ElmFlutterView.cs`.

Reply:
Oh I missed. fixed it.